Bigtable/src/BigtableClient.php

```diff
@@ -69,7 +56,7 @@ class BigtableClient
     const ADMIN_SCOPE = 'https://www.googleapis.com/auth/bigtable.admin';
 
     /**
-     * @var Connection\ConnectionInterface
+     *  @var Google\Cloud\Bigtable\Connection\Grpc
```

Comment:
Why do you change to Grpc? It should stick to the interface whenever possible I think.